docs/individual.rst

```diff
@@ -2,34 +2,38 @@
 Individual element
 ==================
 
-The subject of the Phenopacket is represented by an *Individual* element. Additionally, the *individuals* element is an optional list of Individuals who are related to the subject. The PhenoPackets schema maps the *Individual* element to the FHIR `Patient element <https://www.hl7.org/fhir/patient.html>`_.
+The subject of the Phenopacket is represented by an *Individual* element.
+The PhenoPackets schema maps the *Individual* element to the FHIR `Patient element <https://www.hl7.org/fhir/patient.html>`_.
 
 Individual
 ==========
 This element intends to represent an individual human or other organism. In this documentation, we explain the element using the example of a human proband in a clinical investigation.::
 
   message Individual {
-    string id = 1;
-    string dataset_id = 2; 
-    google.protobuf.Timestamp date_of_birth = 3;
-    Age age_at_collection = 4;
-    OntologyClass sex = 5;
-    repeated Phenotype phenotypes = 6;
-    OntologyClass taxonomy = 7;
-    google.protobuf.Timestamp created = 8;
-    google.protobuf.Timestamp updated = 9;
-    GeoLocation location = 10;
-    Attributes attributes = 11;
-
-    // External identifiers representing this individual. These are considered
-    // different representation of the same record, not records which are in some
-    // other relation with the record at hand.
-    repeated ExternalReference external_identifiers = 12;
+   string id = 1;
+   string dataset_id = 2;
+   google.protobuf.Timestamp date_of_birth = 3;
+   oneof age {
+       Age age_at_collection = 4;
+       AgeRange age_range_at_collection = 5;
+   }
+   Sex sex = 6;
+   KaryotypicSex karyotypic_sex = 10;
+   OntologyClass taxonomy = 8;
   }
 
+
+id
+==
+This element is the primary identifier for the individual. The contents of the element are context dependent, and will
+be determined by the application. For instance, if the Phenopacket is being used to represent a case study about
+an individual with some genetic disease, the individual may be referred to in that study by their position in
+the pedigree, e.g., III:2 for the second person in the third generation. In this case, id would be set to ``III:2``.
```

Comment:
Shouldn't the id in the Individual be the same as that used in the Pedigree, not the position in the pedigree?  e.g. Individual.id == Pedigree.Person.individual_id. This *could* be the position in the pedigree, but this is not mandated.